packages/react-native/scripts/cocoapods/codegen.rb

```diff
@@ -108,3 +108,11 @@ def run_codegen!(
     codegen_utils.generate_react_codegen_podspec!(react_codegen_spec, codegen_output_dir)
   end
 end
+
+def basePath(react_native_path, relative_installation_root)
+  if react_native_path.start_with?("/")
```

Comment:
Another reason for reusing a standard library function, this check will not work for Windows paths.